packages/front-end/components/Features/FeatureModal.tsx

```diff
@@ -98,11 +108,12 @@ export default function FeatureModal({ close, onSuccess }: Props) {
 
   return (
     <Modal
+      inline={inline}
       open={true}
-      size="lg"
```

Comment:
Yea... we definitely do. Currently, if we pass `inline` into the modal, we're setting size to `"fill"`, but otherwise defaulting to `"md"`. I can update this so we're passing the size in explicitly, based off whether or not `inline === true`. Though, I don't like how `inline` is used for different things within the `<Modal />` component. I feel like this is confusing - I'll chew on a better way to handle this.

Reply:
My comment above is now outdated based on the changes we made in the `Modal` component.